frontend/server/src/test/java/org/pytorch/serve/ModelServerTest.java

```diff
@@ -1244,4 +1248,65 @@ private void testMetricManager() throws JsonParseException, InterruptedException
             }
         }
     }
+
+    private void testTSValidPort()
+            throws InterruptedException, GeneralSecurityException, InvalidSnapshotException,
+                    IOException {
+        //  test case for verifying port range refer https://github.com/pytorch/serve/issues/291
+        server.stop();
+        System.setProperty("tsConfigFile", "src/test/resources/config_port.properties");
+        FileUtils.deleteQuietly(new File(System.getProperty("LOG_LOCATION"), "config"));
+        ConfigManager.init(new ConfigManager.Arguments());
+        configManager = ConfigManager.getInstance();
+        PluginsManager.getInstance().initialize();
+
+        InternalLoggerFactory.setDefaultFactory(Slf4JLoggerFactory.INSTANCE);
+        server = new ModelServer(configManager);
+        server.start();
+
+        Channel channel = null;
+        Channel managementChannel = null;
+        for (int i = 0; i < 5; ++i) {
+            channel = TestUtils.connect(false, configManager);
+            if (channel != null) {
+                break;
+            }
+            Thread.sleep(100);
+        }
+
+        for (int i = 0; i < 5; ++i) {
+            managementChannel = TestUtils.connect(true, configManager);
+            if (managementChannel != null) {
+                break;
+            }
+            Thread.sleep(100);
+        }
+
+        Assert.assertNotNull(channel, "Failed to connect to inference port.");
+        Assert.assertNotNull(managementChannel, "Failed to connect to management port.");
+
+        testPing(channel);
+    }
+
+    private void testTSInvalidPort()
+            throws InterruptedException, GeneralSecurityException, InvalidSnapshotException,
+                    IOException {
+        //  test case for verifying port range refer https://github.com/pytorch/serve/issues/291
+        //  invalid port test
+        server.stop();
+        System.setProperty("tsConfigFile", "src/test/resources/config_invalid_port.properties");
```

Comment:
I was thinking about this test a little more. Since this test is only testing hte JAVA code path, couldn't we simply modify the [TS_INFERENCE_ADDRESS](https://github.com/pytorch/serve/blob/master/frontend/server/src/main/java/org/pytorch/serve/util/ConfigManager.java#L52) property in [ConfigManager](https://github.com/pytorch/serve/blob/master/frontend/server/src/test/java/org/pytorch/serve/ModelServerTest.java#L622) and restart the server? Do we need this additional config.properties file?

Reply:
+1

This is a much simpler approach. Thanks.